checks/script_tag.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Check if:
+ * A <script> tag is included in header.php or footer.php
+ */
+class Script_Tag implements themecheck {
+	protected $error = array();
+
+		function check( $php_files, $css_files, $other_files ) {
+
+		$ret = true;
+
+		checkcount();
+		/**
+		 * This check is limited to header.php and footer.php.
+		 */
+		foreach ( $php_files as $file_path => $file_content ) {
+			$filename = tc_filename( $file_path );
+			if ( 'footer.php' === $filename && preg_match( '/<script/', $file_content ) || 'header.php' === $filename && preg_match( '/<script/', $file_content )) {
+				$error         = '/<script/';
+				$grep          = tc_preg( $error, $file_path );
+				$this->error[] = sprintf( '<span class="tc-lead tc-required">' . __( 'REQUIRED', 'theme-check' ) . '</span>: ' . __( 'Found a script tag in %s. Scripts and styles needs to be enqueued or added via a hook, not hard coded.', 'theme-check' ),
```

Comment:
A common thing I'm seeing in Theme Check is that there's often a lack of spaces between error and the examples. This seems simpler and easier to read:

```suggestion
				$this->error[] = sprintf(
					'<span class="tc-lead tc-required">' . __( 'REQUIRED', 'theme-check' ) . '</span>: %s %s',
					sprintf(
						__( 'Found a script tag in %s. Scripts and styles needs to be enqueued or added via a hook, not hard coded.', 'theme-check' ),
						'<strong>' . $filename . '</strong>'
					),
					$grep
				);
```